src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -113,25 +112,25 @@ private static bool IsCodeActionNotSupportedByLSP(IUnifiedSuggestedAction sugges
             var diagnosticsForFix = GetApplicableDiagnostics(request.Context, suggestedAction);
 
             using var _ = ArrayBuilder<LSP.CodeAction>.GetInstance(out var builder);
-            using var _1 = ArrayBuilder<string>.GetInstance(out var codeActionPathList);
+            var codeActionPathList = ImmutableArray.Create(codeAction.Title);
             var nestedCodeActions = CollectNestedActions(request, codeActionKind, diagnosticsForFix, suggestedAction, codeActionPathList, isTopLevelCodeAction: true);
 
             Command? nestedCodeActionCommand = null;
             var title = codeAction.Title;
 
-            var codeActionPath = codeActionPathList.ToArray();
+            var codeActionPathArray = codeActionPathList.ToArray();
```

Comment:
Don't think you need both `codeActionPathList` and `codeActionPathArray` right?  can probably just inline the ToArray on the places that don't accept an immutable array